swagger.yaml

```diff
@@ -5136,6 +5141,8 @@ definitions:
       name:
         type: string
         description: A fully qualified image name (FQIN).
+      identity:
```

Comment:
The name already exists above.  Seems like a duplication.

Reply:
It should be deleted once the new API will be used.